### DISCUSSION TITLE
Networking/hacking for young/early-career open source contributors, especially Orcasound GSoCers!

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orcahello#97] [2022 Microsoft hackathon]
On @valentina-s 's wise suggestion, I'd like to invite past and present [Google Summer of Code contributors to Orcasound](https://www.orcasound.net/portfolio/google-summer-of-code-open-source-software-for-students-orcas/), as well as other early-career contributors to Orcasound's open source efforts, to network with each other and with the Microsoft employees who are volunteering as hackers next week (Mon 9/19 - Wed 9/21). While time zone differences make it hard to have a single synchronous interaction, we hope that Github tools like this Discussion feature will help you all get to know and hack alongside each other a bit!

**You're invited to the 2022 Microsoft hackathon!**

- 2022 GSoC contributors: @karan2704 @Devdoot57 @TGAmbra @Benjamintdk (Karan, Devdoot, Ambra, Benjamin)
- 2022 Beam Reach interns: @emvierling, @tsuize (Emily and Ze)
- 2021 GSoC contributors: @imacchiavello @Molkree @wetdog @DhananjayPurohit (Isabella, Dmitry, Jose, Dhananjay)
- 2020 GSoC contributors: @jd-rs @kunakl07 (Diego, Kunal)

Feel free to just say "hi" to each other in this discussion (we'd love to know what is keeping you busy!), the Microsoft hackers (who will often be working here on _Issues_, coordinating through _Projects_, and adding documentation to the _Wiki_), and/or your mentors...

- Orcasound mentors: @yosoyjay @valentina-s @paulcretu @mcshicks @prafulfillment @veirs @scottveirs @pmdnhd @UXBrendan (Jesse, Valentina, Paul, Steve, Praful, Val, and Scott)

If you'd like to get more involved next week, you can email me at sveirs@gmail.com and I'll work with @cchanke1 to onboard you here and within Microsoft Teams.

**Comment by DhananjayPurohit** [comment]
Hey everyone,
Thanks for the invitation. I would love to hack with you all this season as well. I recently wrapped up my work with [Alby](https://getalby.com/) under [Summer Of Bitcoin](https://www.summerofbitcoin.org/) (an open source program just like GSoC to support the technologies around Bitcoin). You can take a look at what I have built in the last 3 months here (https://github.com/getAlby/lsat-middleware) and read my blogs about it [here](https://dhananjaypurohit.medium.com/building-a-content-distribution-proxy-implementing-the-lsats-spec-summer-of-bitcoin22-at-alby-ae67fcb30a9e) & [here](https://dhananjaypurohit.medium.com/building-a-middleware-library-implementing-the-lsats-spec-summer-of-bitcoin22-at-alby-a64455a62568). Looking forward to meet and hack with the team.

**Reply by scottveirs** [reply]
Super! I've sent a request to @cchanke1 (Chris Hanke) to add you to the Teams workspace!

**Comment by Devdoot57** [comment]
Hi all, I would also love to contribute during this hackathon.

**Reply by scottveirs** [reply]
Hey @Devdoot57 Can you check your email for a Teams invite from @cchanke1 (Chris Hanke)?

**Reply by Devdoot57** [reply]
No, I have not received any email yet.

**Reply by cchanke1** [reply]
@Devdoot57 I may have missed it, but can you reply back with your email address? I need that to invite you to our teams channel.

**Reply by Devdoot57** [reply]
Here you go- devdootchatterjee.x@gmail.com

**Reply by cchanke1** [reply]
It looks like I had already added you. I will try to Tag you from teams now.

**Comment by tsuize** [comment]
Hi guys, I'm Ze. I'm glad to hack with you all! Hope we have a good hackathon experience!

**Comment by karan2704** [comment]
Hello everyone, I am Karan. It is going to be a little difficult for me to contribute during the 2 days given my current schedule but I would still like to receive updates regarding the events of the hackathon.

**Comment by wetdog** [comment]
Hi Team,

I would like to take part and share a great time with you.

Do I have to use this email to set up the teams account? I couldn't access
the team chat and I would need some help to set up my account.

**Reply by cchanke1** [reply]
@wetdog reply back with your email address and I can add you to the Orca Hackathon Teams channel.

**Reply by wetdog** [reply]
Hi Chris, can you add me with joseo913@hotmail.com

**Reply by cchanke1** [reply]
Hi Jose, I just added you. When I add an external guest the messages says "it might take a while", but hopefully it won't be too long.

**Comment by TGAmbra** [comment]
Nice to meet you all, I am Ambra. This is an amazing opportunity and I would have loved to join, but unfortunately I won't be able to join as I am sick these days. I would love to get updates too though.

**Reply by scottveirs** [reply]
> Nice to meet you all, I am Ambra. This is an amazing opportunity and I would have loved to join, but unfortunately I won't be able to join as I am sick these days. I would love to get updates too though.

Hope you feel better soon, Ambra!
